### DISCUSSION TITLE
3D printed case designs?

### DISCUSSION THREAD

**Discussion by traverseda** [STEMinds/STEMinds-Docs#6] [General]
I know there's at least one 3D printed case design, although it sadly doesn't work with the expansion board and I'm not sure if I'm allowed to share it here.

Would it be possible to get some resources (like dimensions, connection location, etc) for making the cases? A 3D model of the PCB would be ideal, and I think many PCB programs let you export one without necessarily exporting the entire board schematic or anything crazy like that.

**Comment by traverseda** [comment]
[eduponics_mini_case.zip](https://github.com/STEMinds/STEMinds-Docs/files/9331977/eduponics_mini_case.zip)

Here is the case I have, I got this from an email and I'm not sure who made is originally.

**Reply by Zontex** [reply]
@traverseda 3D models now available of the hardware, this could help to design your own case for Eduponics mini.

The guy behind the eduponics_mini_case.zip named Bart, good friend of ours :) I'll add it to the git repository as well.

**Reply by traverseda** [reply]
I'll be honest, I don't think I can design as nice of a case as Bart did for the expansion board...

Since he's a good friend, what are the odds he'd be willing to design a case for the expansion board? :p

I will be taking a crack at designing a case for the expansion board myself though, now that there's a 3D model. I just expect it to not be as nice.

**Reply by Zontex** [reply]
I'll let Bart know and if he is busy we will figure a way around this, might need a week or so but will definitely put it as a priority!

**Comment by csthomas2** [comment]
I would be super interested in a case that includes the expansion board. To keep it protected from water, I was able to jam it into the plastic case that ships with the Mini Kit. I cut a couple holes in what would be considered the bottom (because of how its mounted) to run wires through.

Currently makes me hesitant to share any photos of the current setup.

**Comment by Zontex** [comment]
Hi guys @csthomas2  @traverseda,
update coming up soon to crowd supply v2.0 campaign, I hear you! extension boards 3D enclosures will be made.
